### PR TITLE
fix: typo with lt param of iterator

### PR DESCRIPTION
**Pull request by mistakia**
Fix small typo with `lt` param of the iterator method.

~~[ ] Add test to catch this in the future~~

### CODE REVIEW

**Comment by aphelionz** [comment]
How are you feeling about this one @mistakia ? :)

**Comment by mistakia** [comment]
Taking a look at it again it seems like it's only a problem when `next` is an array of more than a single item. 

What's the best way to create such a situation for the tests? would increasing the pointer count from 1 to 2 suffice?

**Comment by aphelionz** [comment]
The test would be nice but I think the biggest thing here is just to run `npm run build` and commit the dist file changes :+1: 

Otherwise LGTM

File: lib/es5/entry.js [hunk 1]
```diff
@@ -31,6 +31,11 @@ var getWriteFormatForVersion = function getWriteFormatForVersion(v) {
 var getWriteFormat = function getWriteFormat(e) {
   return Entry.isEntry(e) ? getWriteFormatForVersion(e.v) : getWriteFormatForVersion(e);
 };
+/*
+ * @description
+ * An ipfs-log entry
+ */
+
```
Comment (aphelionz) [inline lib/es5/entry.js:38]:
```suggestion
/*
 * @description
 * An ipfs-log entry
 */
```

**Review by aphelionz — APPROVED** [review]

**Pull request merged**